[2021-03-09 11:44:21,919] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: hello_world.hello_task 2021-03-08T12:00:00+00:00 [queued]>
[2021-03-09 11:44:21,927] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: hello_world.hello_task 2021-03-08T12:00:00+00:00 [queued]>
[2021-03-09 11:44:21,927] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2021-03-09 11:44:21,927] {taskinstance.py:867} INFO - Starting attempt 1 of 1
[2021-03-09 11:44:21,927] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2021-03-09 11:44:22,029] {taskinstance.py:887} INFO - Executing <Task(PythonOperator): hello_task> on 2021-03-08T12:00:00+00:00
[2021-03-09 11:44:22,031] {standard_task_runner.py:53} INFO - Started process 29537 to run task
[2021-03-09 11:44:22,171] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: hello_world.hello_task 2021-03-08T12:00:00+00:00 [running]> rep-pc
[2021-03-09 11:44:22,181] {python_operator.py:114} INFO - Done. Returned value was: Hello world!
[2021-03-09 11:44:22,274] {taskinstance.py:1048} INFO - Marking task as SUCCESS.dag_id=hello_world, task_id=hello_task, execution_date=20210308T120000, start_date=20210309T144421, end_date=20210309T144422
[2021-03-09 11:44:31,831] {logging_mixin.py:112} INFO - [2021-03-09 11:44:31,830] {local_task_job.py:103} INFO - Task exited with return code 0
